flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/functions/hive/HiveGenericUDTFTest.java

```diff
@@ -238,4 +242,28 @@ public void process(Object[] args) throws HiveException {
         @Override
         public void close() {}
     }
+
+    /** Test split udtf initialize with StructObjectInspector. */
+    public static class TestSplitUDTFInitializeWithStructObjectInspector extends GenericUDTF {
```

Comment:
Move it to package org.apache.flink.table.functions.hive.util.

Reply:
done